src/containers/profiles/ViewProfiles/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import M from 'materialize-css';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import avatar2 from '../../../assets/images/avatar2.png';
+import { getUserProfileAction } from '../store/profileActions';
+import ROUTES from '../../../utils/routes';
+import EditProfiles from '../EditProfiles';
+import './ViewProfiles.scss';
+
+class ViewProfiles extends Component {
+  modal = React.createRef();
+
+  componentDidMount = () => {
+    const modals = document.querySelector('.modal');
+    M.Modal.init(modals, {});
+  }
+
+  componentWillMount = () => {
+    const { getUserProfile } = this.props;
+    getUserProfile('bevkololi1');
```

Comment:
Hey @bevkololi ,
I hope you'll replace the argument in the `getUserProfile` method to be dynamic instead of a hard-coded value.

Reply:
I definitely will, this was here just because the signup and registration were not available. Once they are I can be able to get the user.